WcaOnRails/lib/advance_to_next_round_requirement.rb

```diff
@@ -1,22 +1,35 @@
 # frozen_string_literal: true
 
 class AdvanceToNextRoundRequirement
+  include ActiveModel::Validations
```

Comment:
Ok, I attempted to do that in https://github.com/jfly/worldcubeassociation.org/commit/8b5d40fd89ae19ee6cb90532985e2dbf22af7e51. What do you think? I was actually pretty pleased with it until I saw "113 additions and 68 deletions". Now I'm unsure what's better...